### ISSUE TITLE
Visual commands in print preview

### ISSUE THREAD

**Issue by simonkurka** [inkstitch/inkstitch#556]
Hi,
in print preview the visual commands show up. Most of the time only one quarter of them.
Is it intended that they show up in print preview?

Occurs in 1.26.2
Not present in 1.25.0 if there's a layer as top-level object, otherwise it looks quite the same.

I really liked it when they were not printed. That's why im asking if it's by intention ;-)

Is there a way to hotfix this some kind?

Thank you in advance!

**Comment by lexelby** [comment]
It's not by intention!  Thank you for catching this.  Unfortunately, I don't know of a workaround for this bug.

**Comment by simonkurka** [comment]
Seems to be OS independent. Tested with Windows and Linux.

**Comment by simonkurka** [comment]
https://github.com/inkstitch/inkstitch/blob/master/print/resources/style.css
```CSS
figure.inksimulation svg g[id^="group"] {
  display: none;
}
```
Adding this hides them completely. **But there must be a far better solution** :-/

I tried to catch the visual commands by `g[inkscape|label^="Ink/Stitch Command"]`, but that seems to be quite complicated due to the presence of the inkscape-namespace.

**Comment by lexelby** [comment]
I'll try to get an actual fix out together soon.  I'm sorry this is causing you so much trouble!

It occurs to me that there may be another bug that may help.  Try toggling the visibility of the command groups in Inkscape.  I think Ink/Stitch might obey commands every if they're set to display:none in the svg.

**Comment by simonkurka** [comment]
This is no trouble comparing to not having Ink/Stitch. Thank you for the great work!
I like open source an I'm happy to be able to investigate some things.

If you have a fix out, it would be great if you link the commit. I'm really interested to it.

**Comment by lexelby** [comment]
Here's what happened: in a recent version, we improved the input extension significantly.  If you open an embroidery file (such as a DST) with built-in trims, they will show up as visual commands.

Unfortunately, the same code that renders a DST or other embroidery file into an SVG is also used to render the print preview.  When I added the new feature, I forgot all about print preview -- until you noticed :)
